.github/workflows/pronto.yml

```diff
@@ -0,0 +1,26 @@
+name: Pronto
+on: ['push', 'pull_request']
+
```

Comment:
We should limit this job to only run when ruby related files have changed. Perhaps use the same conditions as main_build uses?

Alternatively, this could be put into the main build.

Reply:
Great! I'll add to the main_build

Reply:
Actually, I think it would be better to run in another job, because we'd probably want to know when both are failing.

Reply:
In theory, I agree. Unfortunately, it takes a really long time for new jobs to boot as it relates to loading the cache and installing any missing packages. Ideally, everything that can run in parallel should but it's not really practical.